src/library/export/trackexportdlg.cpp

```diff
@@ -68,29 +68,30 @@ void TrackExportDlg::slotAskOverwriteMode(
             QMessageBox::Warning,
             tr("Overwrite Existing File?"),
             tr("\"%1\" already exists, overwrite?").arg(filename),
-            QMessageBox::Cancel | QMessageBox::No | QMessageBox::NoToAll
-            | QMessageBox::Yes | QMessageBox::YesToAll);
-    question_box.setDefaultButton(QMessageBox::No);
-    question_box.addButton(tr("&Overwrite"), QMessageBox::YesRole);
-    question_box.addButton(tr("Over&write All"), QMessageBox::YesRole);
-    question_box.addButton(tr("&Skip"), QMessageBox::NoRole);
-    question_box.addButton(tr("Skip &All"), QMessageBox::NoRole);
+            QMessageBox::Cancel);
 
-    switch (question_box.exec()) {
-    case QMessageBox::No:
+    QPushButton* pSkip = question_box.addButton(
+            tr("&Skip"), QMessageBox::NoRole);
+    QPushButton* pSkipAll = question_box.addButton(
+            tr("Skip &All"), QMessageBox::NoRole);
+    QPushButton* pOverwrite = question_box.addButton(
+            tr("&Overwrite"), QMessageBox::YesRole);
+    QPushButton* pOverwriteAll = question_box.addButton(
+            tr("Over&write All"), QMessageBox::YesRole);
+    question_box.setDefaultButton(pSkip);
+
+    question_box.exec();
+    auto pBtn = question_box.clickedButton();
```

Comment:
clang-tidy is suggesting: 
```suggestion
    auto* pBtn = question_box.clickedButton();
```

Reply:
:+1: